gateway/server.go

```diff
@@ -137,7 +137,8 @@ func main() {
 	if config.ScaleFromZero {
 		scalingConfig := scaling.ScalingConfig{
 			MaxPollCount:         uint(1000),
-			FunctionPollInterval: time.Millisecond * 10,
+			SetScaleRetries:      uint(20),
```

Comment:
`SetScaleRetries` sounds like it should be a function, perhaps this should just be `MaxScaleRetries`

Reply:
`MaxScale` seems to have some ambiguity with the maximum replicas/scale allowed for a function. `SetScale` is referring to the upstream call. Do you think there's another name that can fit here?

Reply:
`MaxScaleAttempts` maybe

Reply:
MaxScale also has the same connotation as before, max scale is a label we set for controlling scaling. If `ScaleUpAttempts` seems better to you I could change to that?

Reply:
I can't think of anything that is clearly better so we might as well leave it alone